src/documentation/pages/components/forms/range-slider/examples/_base_consistent.vue

```diff
@@ -0,0 +1,62 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base-consistent" :tabs="exampleTabs">
+    .chi-form__item(slot="example")
+      label(class="chi-label", id="range-label", for="range04") Range label
+      .chi-input__wrapper
+        input(class="chi-range-slider", type="range", ref="range")
```

Comment:
On label you are using  `for="range04"`
```suggestion
        input(class="chi-range-slider", type="range", ref="range" id="range04")
```